src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/FilterConfiguration.kt

```diff
@@ -15,6 +15,11 @@ class FilterConfiguration {
   fun chooseFilter(): FilterRegistrationBean<ChooseSupplierFilter> =
     FilterRegistrationBean<ChooseSupplierFilter>().apply {
       this.filter = ChooseSupplierFilter()
-      this.addUrlPatterns(DASHBOARD_URL, SELECT_MONTH_URL, JOURNEYS_URL, MOVES_BY_TYPE_URL)
+      this.addUrlPatterns(
+        *HtmlController.routes(),
```

Comment:
I really don't like the name of the this **HtmlController**.  It tells me nothing....